src/build.ts

```diff
@@ -35,8 +35,8 @@ export async function copyPublicAssets (nitro: Nitro) {
 }
 
 export async function build (nitro: Nitro) {
-  const rollupConfig = getRollupConfig(nitro)
   await nitro.hooks.callHook('rollup:before', nitro)
+  const rollupConfig = getRollupConfig(nitro)
```

Comment:
I think put hook before to allow extending resolved config but not possible. We can use new hook for that :)

Reply:
Makes sense 👍